### DISCUSSION TITLE
Extension of "useCart" hook

### DISCUSSION THREAD

**Discussion by lastmaj** [saleor/saleor#8563] [Q&A]
Hello, 
this topic had been posted on [Spectrum](https://spectrum.chat/saleor/development/extension-of-usecart-hook~06cd6268-3e64-40bd-a106-401deede5e83), but no answers were given. 
I want to achieve the same thing; I want to add some fields to the items present in the cart. How could I access the code of useCart and extend it ? 
thanks in advance

**Comment by karolkielecki** [comment]
Our latest SDK no longer provides client-side cart.  We recommend using Checkout API on the Saleor's side.
